app/controllers/api/v0/swagger_models.rb

```diff
@@ -101,6 +93,39 @@ module Api::V0::SwaggerModels
     end
   end
 
+  swagger_schema :Presentation do
+    key :required, [:content_uid, :user_uid, :presented_at]
+    property :content_uid do
+      key :type, :string
+      key :description, "The a15k ID of the content being presented."
+    end
```

Comment:
for fields that are uuid, I think we should set the format https://swagger.io/specification/#dataTypes
```
  key :format, :uuid
```